docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-credential-update.md

```diff
@@ -12,6 +12,79 @@ Currently EKS Anywhere does not support updating vSphere credential when upgradi
     - `vsphere-credentials` under `eksa-system` namespace - Update `password`, `passwordCP`, `passwordCSI` field under data.
     - If annotation `kubectl.kubernetes.io/last-applied-configuration` exists on any of the above Secret object, update password in `kubectl.kubernetes.io/last-applied-configuration` field.
     -  `{CLUSTER_NAME}-csi-vsphere-config` under `eksa-system` namespace - If annotation `kubectl.kubernetes.io/last-applied-configuration` exists on the secret object, update password in `kubectl.kubernetes.io/last-applied-configuration` field.
+- Alternatively you can use the vSphere credential update script below to update vSphere credential. Before running the script, please make sure that
```

Comment:
can we have this as a separate option? Like option1 - Manual steps, option2 - script that you created, option3 - run upgrade?